README.md

```diff
@@ -91,7 +91,25 @@ For the `edit` role, in addition to the permissions available to `view`:
 
 Users authenticated against OpenShift OAuth will be added to the Jenkins authorization matrix upon their first successful login.
 
-Permissions for users in Jenkins can be changed in OpenShift after those users are initially established in Jenkins.  The OpenShift Login plugin polls the OpenShift API server for permissions and will update the permissions stored in
+Now, with v1.0.10 of this plugin, you can change
+
+* Which OpenShift Roles are checked for authorization
+* Which Jenkins permissions map to which OpenShift Roles
+
+This plugin will look at for a ConfigMap named openshift-jenkins-login-plugin-config.  If it finds and can read in that ConfigMap, it then:
+
+* Checks the `data` map for a key named `roleRanking`
+* If found, that defines the list of OpenShift Roles to perform authorization review against.
+* The order of the list matters. The first entry should have the most permissions in Jenkins. So it is the equivalent of the `admin` role with our default settings.
+* The authorization reviews are considered complete with the first successful result.  So, just like with our defaults, the most authorized role is used.
+* Any other key/value pair is consider a Jenkins permission to OpenShift Role mapping.
+* The key is the Jenkins permission group short ID and the Jenkins permission short ID, with those two separated by a hyphen or `-` character.
+* So if you want to add the Overall Jenkins Administer permission to an OpenShift Role, the key should be `Overall-Administer`
+* To get a sense of which permission groups and permissions IDs are available, go the the matrix authorization page in the Jenkins console and IDs for the groups and individual permissions in the table they provide.
+* The value of the key/value pair is the list of OpenShift Roles the permission should apply to, with each role separated by a comma or `,`.
+* So if you want to add the Overall Jenkins Administer permission to say both the default `admin` and `edit` roles, as well as a new `jenkins` role you have created, the value for the key `Overall-Administer` would be `admin,edit,jenkins`.    
```

Comment:
> I also wonder if it would be better to check all the roles, rather than stopping on the first one.

I'm having difficulty putting my finger on a specific doc saying it but I'm under the impression that OpenShift honors the aggregate of role bindings and would expect this to do the same as long as the permissions granted are additive.  We should ensure there is no inconsistency with permission models else there will be confusion and problems.